### ISSUE TITLE
🐞 Bug: can't install on my new Acer ChromeBook

### ISSUE THREAD

**Issue by antoine2711** [chromebrew/chromebrew#8168]
**Describe the bug**
When installing, at one point, I get this:

```shell
Zip archive downloaded.
Unpacking archive using 'tar', this may take a while...
Performing pre-install...
Performing install...
Performing post-install for glibc...
Package glibc version is 2.32.
System glibc version is 2.32.
Creating symlinks to system glibc version to prevent breakage.
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
Performing post-install for crew_profile_base...
Appending `source /usr/local/etc/profile` to the beginning of /home/chronos/user/.bashrc...
Performing post-install for gcc...
Performing post-install for icu4c...
Finding Packages expecting icu4c version 67 that may need updating:
Finding Packages expecting icu4c version 68 that may need updating:
Finding Packages expecting icu4c version 69 that may need updating:
Finding Packages expecting icu4c version 71 that may need updating:
Performing post-install for libunbound...
/usr/local/sbin/unbound-anchor: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
/usr/local/sbin/unbound-anchor: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
/usr/local/lib/crew/lib/package.rb:305:in `system': Command failed with exit 127: bash
libunbound failed to install: `CFLAGS="-O2 -pipe -ffat-lto-objects -fPIC  -fuse-ld=mold  -flto=auto" CXXFLAGS="-O2 -pipe -ffat-lto-objects -fPIC  -fuse-ld=mold  -flto=auto" FCFLAGS="-O2 -pipe -ffat-lto-objects -fPIC  -fuse-ld=mold  -flto=auto" FFLAGS="-O2 -pipe -ffat-lto-objects -fPIC  -fuse-ld=mold  -flto=auto" LDFLAGS="-flto=auto " bash -c /usr/local/sbin/unbound-anchor -a '/usr/local/etc/unbound/root.key' || /usr/local/sbin/unbound-anchor -4 -a '/usr/local/etc/unbound/root.key'` exited with 127
chronos@localhost / $ curl -Ls git.io/vddgY | bash
curl: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
chronos@localhost / $ file: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
bash: file:: command not found
chronos@localhost / $  export LD_LIBRARY_PATH=/usr/lib:/usr/local/lib 
chronos@localhost / $ curl -Ls git.io/vddgY | bash
chronos@localhost / $ crew
ruby: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
chronos@localhost / $ crew install glibc
```

**To Reproduce**

1. I executed this command:

      ```shell
      curl -Ls git.io/vddgY | bash
      ```

      Output:

      See above.

**Expected behavior**
That it installs.

**Environment: Output of `crew sysinfo -v`**

***
chronos@localhost / $ crew sysinfo -v 
ruby: symbol lookup error: /usr/local/lib64/libpthread.so.0: undefined symbol: __shm_get_name, version GLIBC_PRIVATE
***

**Comment by antoine2711** [comment]
Note that `rm -Rf /usr/local/` and reinstalling get me to the same point.

Regards, A.

**Comment by antoine2711** [comment]
Some system informations:



CHROME VERSION |   | 92.0.4515.162
-- | -- | --
CHROMEOS_ARC_ANDROID_SDK_VERSION |   | 28
CHROMEOS_ARC_POLICY |   |  
CHROMEOS_ARC_POLICY_COMPLIANCE_REPORT |   |  
CHROMEOS_ARC_STATUS |   | enabled
CHROMEOS_ARC_VERSION |   | 7611540
CHROMEOS_AUSERVER |   | https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID |   | {495DCB07-E19A-4D7D-99B9-4710011A65B1}
CHROMEOS_CANARY_APPID |   | {90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER |   |  
CHROMEOS_FIRMWARE_VERSION |   | Google_Nami.10775.145.0
CHROMEOS_RELEASE_APPID |   | {495DCB07-E19A-4D7D-99B9-4710011A65B1}
CHROMEOS_RELEASE_BOARD |   | nami-signed-mp-v5keys
CHROMEOS_RELEASE_BRANCH_NUMBER |   | 88
CHROMEOS_RELEASE_BUILDER_PATH |   | nami-release/R92-13982.88.0
CHROMEOS_RELEASE_BUILD_NUMBER |   | 13982
CHROMEOS_RELEASE_BUILD_TYPE |   | Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE |   | 92
CHROMEOS_RELEASE_DESCRIPTION |   | 13982.88.0 (Official Build) stable-channel nami
CHROMEOS_RELEASE_KEYSET |   | mp-v5
CHROMEOS_RELEASE_NAME |   | Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER |   | 0
CHROMEOS_RELEASE_TRACK |   | stable-channel
CHROMEOS_RELEASE_UNIBUILD |   | 1
CHROMEOS_RELEASE_VERSION |   | 13982.88.0
CLIENT_ID |   | error: consent not given
DEVICETYPE |   | CHROMEBOOK
ENTERPRISE_ENROLLED |   | Not managed
GOOGLE_RELEASE |   | 13982.88.0
HWID |   | AKALI C5B-A5C-E3K-42Q-A6S
LOGDATE |   | Mon Apr 10 16:29:12 UTC 2023 Mon Apr 10 12:29:12 EDT 2023
[CHROME VERSION](chrome://system/#CHROME%20VERSION) |   | 92.0.4515.162

**Comment by uberhacker** [comment]
Thank you for the bug report @antoine2711.  That is an older ChromeOS release.  Have you tried to update to a newer release or has this Chromebook reached EOL?

**Comment by uberhacker** [comment]
I'm guessing you have an Acer Chromebook Spin 13?  If that is the case, I think your EOL is not until June 2025?  See https://support.google.com/chrome/a/answer/6220366#zippy=%2Cacer.  You might try to update to a new release.  See https://www.thegeekstuff.com/2017/08/chrome-os-update-to-latest-version/.

**Comment by antoine2711** [comment]
> Thank you for the bug report @antoine2711. That is an older ChromeOS release. Have you tried to update to a newer release or has this Chromebook reached EOL?

(For background, this is a refurbished machine I just received a few hours ago…)
After finishing the updates for the system that came in, I got this:

```
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 64.5/64.5 kB 2.4 MB/s eta 0:00:00
Installing collected packages: wheel
  Attempting uninstall: wheel
    Found existing installation: wheel 0.38.4
    Uninstalling wheel-0.38.4:
      Successfully uninstalled wheel-0.38.4
Successfully installed wheel-0.40.0
Core installed!

Running Bootstrap package postinstall scripts...

Package crew_mvdir pixz ca_certificates ruby openssl is not installed. :(
Synchronizng local package repo...
Cloning into '/usr/local/lib/crew'...
remote: Enumerating objects: 11, done.
remote: Counting objects: 100% (11/11), done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 11 (delta 0), reused 6 (delta 0), pack-reused 0
Receiving objects: 100% (11/11), 57.49 KiB | 57.49 MiB/s, done.
remote: Enumerating objects: 2115, done.
remote: Counting objects: 100% (2115/2115), done.
remote: Compressing objects: 100% (1856/1856), done.
remote: Total 2115 (delta 272), reused 723 (delta 257), pack-reused 0
Receiving objects: 100% (2115/2115), 1.64 MiB | 956.00 KiB/s, done.
Resolving deltas: 100% (272/272), done.
Updating files: 100% (2116/2116), done.
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at ec1db20 Fix errors in fake packages (#8169)

                       . .
                   ..,:;;;::'..
                 .':lllllllool,.
                ...cl;..... ,::;'.
               .'oc...;::::..0KKo.
               .'od: .:::::, lolc.
             .'lNMMMO ;ooc.,XMMWx;:;.
            .dMMMMMMXkMMMMxoMMMMMMMMO.
            .:O0NMMMMMMMMMM0MMMMMN0Oc.
              .:xdloddddddoXMMMk:x:....
              .xMNOKX0OOOOxcodlcXMN0O0XKc.
              .OMXOKXOOOOOk;ol:OXMK...;N0.
              'XMKOXXOOOOOk:docOKMW,  .kW;
             .cMMKOXXOOOOOOOOOOO0MM;  .lMc.
             .cMM00XKOOOOkkkkkkOOWMl. .cMo.
             .lMWO0XKOOOkkkkkkkkONMo.  ;Wk.
             .oMNO0X0OOkkkkkkkkkOXMd..,oW0'
             .xMNO0X0OOkkkkkkkkkkXMWKXKOx;.
             .0MXOOOOOOkkkkkkkkkOKM0..
             'NMWNXXKK000000KKXNNMMX.
             .;okk0XNWWMMMMWWNKOkdc'.
                .....'cc:cc:''...
  ___ _                               _
 / (_)|\                              |\
|     ||__    ,_    __  _  _  _    __ |/_  ,_    __  _   _   _
|     |/  |  /  |  /  \/ |/ |/ |  |_/ |  \/  |  |_/ /|   |   |\_
 \___/|   |_/   |_/\__/  |  |  |_/|__/\__/   |_/|__/  \_/ \_/


Edit /usr/local/etc/env.d/03-pager to change the default PAGER.
more is used by default

You may wish to edit the /usr/local/etc/env.d/02-editor file for an editor default.

Chromebrew provides nano, vim and emacs as default TUI editor options.
Chromebrew installed successfully and package lists updated.
chronos@localhost / $  
```

Thanks, and I will close this issue as it is solved for me. Thanks for you support and you great work with this tool!

Best Regards,
   Antoine

**Comment by uberhacker** [comment]
Great to hear.  Thanks for your interest in this project.
